### ISSUE TITLE
TitleDB stopped updating?

### ISSUE THREAD

**Issue by proletariado** [wowsuchdoge/doge#6]
Doge says the database is up to date, but it definitely isn't. It's missing River City Girls and the newest smash at the very least, and I'd wager a lot more.

**Comment by dhaos21** [comment]
"NUT_TITLEDB_URL": "https://raw.githubusercontent.com/blawar/titledb/master/"

**Comment by proletariado** [comment]
Thanks. One more piece of advice for anyone who lands here: you may need to rename "US.titles.json" to "titles.json" (and replace any existing file by that name, obviously), as the new source for titledb doesn't seem to use DOGE's naming convention for that particular file.

**Comment by mancupo** [comment]
The best way? I've got it ;-)

Go here: https://github.com/blawar/titledb
Download the zip file there, rename your old titledb folder inside nut to titledb_old (or whatever).
Rename the new one that you just got, from titledb-master to titledb, make sure your switch archive drive is on, do start.bat inside the CMD window, and Doge will know what to do.
I wait and see that Doge has finished in the CMD window before I open a Firefox tab to see what games/updates I'm missing.
Once you're satisfied with the results, you can delete your titledb_old folder.
Titledb folder should be around 364MB after Doge finishes its work.

New updates will appear immediately.  To see new base games, I need to tell Doge to display all Wanted games and then click on the ones I want to follow on.  Then, I tell Doge to display only what I usually want to see (just those new base games I'm watching for (in red) and all the updates I'm missing (in orange).

Doing this will let you keep a perfect set.  Enjoy!

**Comment by proletariado** [comment]
Well, here we are again. Blawar's repos were taken down.

**Comment by dhaos21** [comment]
Yep and now doge is dead because even if you use a diff titleid address, it
still relies on the blawar github.

**Comment by proletariado** [comment]
I hope someone with the know-how forks this and updates it.

**Comment by proletariado** [comment]
Well, for whomever ends up here looking for help:

I have my doge installation working by editing **doge.config.json** to set  **"NUT_TITLEDB_URL": "",** (editing out the original url to fetch the titledb).

I edited **start.sh** to include a couple lines:

```
#!/bin/sh

curl https://tinfoil.media/repo/db/titles.json > ./nut/titledb/titles.json
curl https://tinfoil.media/repo/db/versions.json > ./nut/titledb/versions.json

start http://127.0.0.1:6093

cd server
node server.js
```

Using this edited start.sh doge is working normally for me at present. It downloads titles.json and versions.json from the tinfoil website, it starts the server and opens my default browser, shows me my games, missing dlc/updates, etc.

**Comment by dhaos21** [comment]
doesnt work for me, its having trouble with the versions part as in not updating

**Comment by proletariado** [comment]
bob version cache or versions.json?

**Comment by dhaos21** [comment]
Versions.json or txt never gets downloaded, I can get it working by adding
the file manually.

**Comment by dhaos21** [comment]
blawars repo is back up for those of you that didnt have access to other titledb meta data.
